pombola/south_africa/urls.py

```diff
@@ -159,6 +159,11 @@
             url='/mp-attendance/',
             permanent=False)
     ),
+    url(r'^attendance/$',
```

Comment:
We're not using APPEND_SLASH on this project, so it's best to make the trailing slash optional, I think.